server/config.ts

```diff
@@ -76,14 +76,19 @@ type Debug = {
 	raw: boolean;
 };
 
+type SqliteConfig =
+	| {
+			maxDaysHistory: number | undefined;
+	  }
+	| undefined;
+
```

Comment:
that type seems odd to me?

You shouldn't set that to an object or undefined, things that want it optional shall say so when they mean it.

Meaning, the user of the type should do 
```typescript
export type ConfigType = {
 // ...
sqliteConfig?: SqliteConfig;
// ...
}
```
Or whatever, but not that funny type that is intrinsically this way.
That's a hassle to use